server.js

```diff
@@ -9,10 +13,21 @@ const accountRouter = require('./routes/account.router');
 const savingRouter = require('./routes/saving.router');
 const statsRouter = require('./routes/stats.router');
 
+const { jwtCallback } = require('./config/passport');
+
 const app = express();
 
 app.use(express.json());
 
+app.use(passport.initialize());
+
+const opts = {
+  jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+  secretOrKey: process.env.JWT_SECRET,
+};
+
+passport.use(new JwtStrategy(opts, jwtCallback));
+
 app.use('/categories', categoryRouter);
```

Comment:
Probably you should protect all endpoints by `isAuth` guard?

Reply:
Yeah, thanks for reminder I will do it